README.md

```diff
@@ -106,23 +112,31 @@ Run the following commands, consequently:
     feature-flags    	standard     	Feature Flags service for controlling feature rollout
     ...
 
+**NOTE:**  If you're using a trial account, the service plan will be `lite`.
```

Comment:
Add `lite` on line 112.
"the service plan will be `lite`" -> "only the `lite` plan is available"